styles/pronouns.cc/catppuccin.user.css

```
@@ -0,0 +1,251 @@
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+    a {
+      --bs-link-color-rgb: #rgbify(@blue) [] !important;
+    }
+    --bs-link-color-rgb: #rgbify(@blue) [];
+    --bs-link-hover-color: @sapphire;
+    --bs-body-color: @text;
+    --bs-body-bg: @crust;
+    --bs-darkk-rgb: #rgbify(@base);
+    --bs-lightt-rgb: #rgbify(@base);
+    .bg-light,
+    :root[data-bs-theme="light"] .bg-light {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-darkk-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+    }
+    .bg-dark {
+      --bs-bg-opacity: 1;
+      background-color: rgba(
+        var(--bs-lightt-rgb),
+        var(--bs-bg-opacity)
+      ) !important;
+    }
+    --bs-emphasis-color: @accent-color;
+    --bs-emphasis-color-rgb: #rgbify(@accent-color) [];
+    --bs-secondary-color: rgba(222, 226, 230, 0.75);
+    --bs-secondary-color-rgb: @subtext0;
+    --bs-secondary-bg: @base;
+    --bs-secondary-bg-rgb: #rgbify(@base) [];
+    --bs-tertiary-color: rgba(222, 226, 230, 0.5);
+    --bs-tertiary-color-rgb: 222, 226, 230;
+    --bs-tertiary-bg: @surface0;
+    --bs-tertiary-bg-rgb: #rgbify(@surface0) [];
+    --bs-primary-text-emphasis: @accent-color;
+    --bs-secondary-text-emphasis: @subtext1;
+    --bs-success-text-emphasis: @green;
+    --bs-info-text-emphasis: @teal;
+    --bs-warning-text-emphasis: @yellow;
+    --bs-danger-text-emphasis: @red;
+    --bs-light-text-emphasis: @subtext0;
+    --bs-primary-bg-subtle: @mantle;
+    --bs-secondary-bg-subtle: @base;
+    --bs-primary-border-subtle: @blue;
+    --bs-secondary-border-subtle: @overlay0;
+    --bs-success-border-subtle: @green;
+    --bs-info-border-subtle: @teal;
+    --bs-warning-border-subtle: @yellow;
+    --bs-danger-border-subtle: @red;
+    --bs-light-border-subtle: @overlay1;
+    --bs-dark-border-subtle: @surface1;
+    --bs-heading-color: inherit;
+    --bs-link-color: @blue;
+    --bs-link-hover-color: @blue;
+
+    --bs-code-color: @pink;
+    --bs-highlight-color: @rosewater;
+    --bs-highlight-bg: #664d03;
+
+    // navbar
+    .navbar {
+      --bs-navbar-color: rgba(#rgbify(@text) [], 0.55) !important;
```

You can just use Less functions here like [`fade`](https://lesscss.org/functions/#color-operations-fade).
```suggestion
      --bs-navbar-color: fade(@text, 55%) !important;
```

---

(Do the same for the below two lines and any others like this.)

---

think i got all of the rgba which needed it